README.md

```diff
@@ -2,57 +2,73 @@
 
 ![](https://github.com/oam-dev/rudr/workflows/Rust/badge.svg)
 
-[Open Application Model (OAM)](https://github.com/oam-dev/spec/blob/master/1.purpose_and_goals.md) is a portable application definition which provides a platform-agnostic approach to describe cloud and edge applications. OAM's philosophy is building application centric infrastructure which let users choose where to run application based on **application level concerns**. In order for this model to work, a runtime needs to implement [OAM spec](https://github.com/oam-dev/spec).
+Rudr is an [Open Application Model (OAM)](https://github.com/oam-dev/spec/blob/master/1.purpose_and_goals.md) implementation for Kubernetes. 
 
-Rudr is the reference OAM implementation for Kubernetes. 
+***Note: Rudr is currently in alpha. It may reflect the API or features we are vetting before inclusion into the OAM spec.***
 
-> Note: Rudr is currently in alpha. It may reflect the API or features we are vetting before inclusion into the Open App Model spec.
+## Why should I care?
 
-## We believe creating cloud-native applications should not be hard
+Kubernetes API resources focused on container infrastructure rather than the applications themselves. Yet users want to think in terms of application architecture, not of infrastructure.
 
-Modeling, deploying and operating applications shouldn’t have to claim complex infrastructure configurations, and we hope application maintainers could think in terms of application architecture, not of infrastructure.
+**Rudr provides application level primitives for Kubernetes that enable:**
 
-However, achieving this directly with Kubernetes is hard. 
+- The ability to define application (e.g., WordPress) in Kubernetes. 
+- The ability to define operational capability (e.g., auto-scaling policy, rather than HPA) in Kubernetes.
+- A standard and portable description of Kubernetes application (e.g., a infrastructure agnostic and self-descriptive application definition, includes every dependency and operational ability the application requires to run).  
+- An application centric abstraction atop container infrastructure.
 
-At the heart of it, container orchestration systems inextricably mixes application primitives with infrastructure primitives. Different roles, such as developers and operators, have to concern themselves with problems from each other's domain as well as learning infrastructure details in order to understand the whole picture of the deployment.
+**Rudr can be used by:**
```

Comment:
I think the "used by" sounds confusing.
Maybe just call it **Rudr use cases** or **user scenarios**.